student.rb

```diff
@@ -1,6 +1,7 @@
-require './classroom'
+require_relative './classroom'
+require_relative './person'
 class Student < Person
-  attattr_accessor :@classroom
+  attattr_accessor :classroom
 
```

Comment:
Same here, the `attr_accessor` is misspelled with `attattr_accessor`. kindly fix that to prevent errors when utilizing the getter and setter for `: classroom`.

This is causing the following error:

![image](https://user-images.githubusercontent.com/38088818/226102701-07bcb743-4477-41b1-903f-b9f524950bad.png)